tests/e2e-pw/specs/shopper/shopper-subscriptions-purchase-free-trial.spec.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import test, { expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { shouldRunSubscriptionsTests } from '../../utils/constants';
+import { describeif, getMerchant, getShopper } from '../../utils/helpers';
+import RestAPI from '../../utils/rest-api';
+import { config } from '../../config/default';
+import {
+	confirmCardAuthentication,
+	emptyCart,
+	fillCardDetails,
+	setupCheckout,
+} from '../../utils/shopper';
+import {
+	goToCart,
+	goToProductPageBySlug,
+} from '../../utils/shopper-navigation';
+import { goToOrder, goToSubscriptions } from '../../utils/merchant-navigation';
+
+const nowLocal = new Date();
+const nowUTC = new Date(
+	nowLocal.getUTCFullYear(),
+	nowLocal.getUTCMonth(),
+	nowLocal.getUTCDate()
+);
+const formatter = new Intl.DateTimeFormat( 'en-US', {
+	dateStyle: 'long',
+} );
+const renewalDate = nowUTC.setDate( nowUTC.getDate() + 14 );
+const renewalDateFormatted = formatter.format( renewalDate );
+const productName = 'Subscription free trial product';
+const productSlug = 'subscription-free-trial-product';
+const customerBilling = config.addresses.customer.billing;
+const customerBillingConfig =
+	config.addresses[ 'subscriptions-customer' ].billing;
+let orderId, subscriptionId;
+
+const testSelectors = {
+	productSubscriptionDetails: '.summary span.subscription-details',
+	cartSubscriptionDetails: '.product-price span.subscription-details',
+	cartSubscriptionFirstPaymentDate: '.first-payment-date',
+	cartOrderTotal: 'tr.order-total:not(.recurring-total) td',
+	checkoutSubscriptionDetails: 'span.subscription-details',
+	checkoutSubscriptionFirstPaymentDate: '.first-payment-date',
+	checkoutPlaceOrderButton: '#place_order',
+	checkoutOrderId: '.woocommerce-order-overview__order.order > strong',
+	checkoutSubscriptionId: 'td.subscription-id > a',
+	wcOrderPaymentId: '.woocommerce-order-data__meta',
+	subscriptionStatus: '.subscription-status',
+	subscriptionProductName: '.order-item',
+	subscriptionRecurringTotal: '.recurring_total',
+	subscriptionTrialEnd: 'time.trial_end_date',
+};
+
+describeif( shouldRunSubscriptionsTests )(
+	'Shopper: Subscriptions - Purchase Free Trial',
+	() => {
+		test.beforeAll( async ( {}, { project } ) => {
+			const restApi = new RestAPI( project.use.baseURL );
+			await restApi.deleteCustomerByEmailAddress(
+				customerBillingConfig.email
+			);
+		} );
+
+		test( 'Merchant should be able to purchase a free trial', async ( {
+			browser,
+		} ) => {
+			const { shopperPage } = await getShopper( browser );
```

Comment:
Since you're deleting `subscriptions-customer`, it’s better to use `getAnonymousShopper()` instead, as `getShopper()` logs in with a different user (the regular customer).

Reply:
Actually, this test (and IMO most of the tests) doesn't need deleting the customer, we can go on with the regular customer. I just copied what was there in the puppeteer test. What about we deleting the regular customer instead to reset the saved cards? Did you understand the reason why we're deleting the customers?

Reply:
@eduardoumpierre any follow-ups to my above statement?

Reply:
Ok, just realized that the test doesn't need customer deletion, changed the process to use the standard customer. You were right about the first comment by the way, it didn't have any effect on the regular shopper. Thanks for bringing that up!